Xamarin.Forms.Platform.Android/ITabStop.cs

```diff
@@ -0,0 +1,9 @@
+﻿using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	internal interface ITabStop
```

Comment:
This interface _does_ need to be public (same as iOS); see the discussion with @programmation. My apologies for the confusion.